src/app/settings/edit-account/page.tsx

```diff
@@ -6,16 +6,121 @@ import ActiveButton from '@/components/Active-Button/ActiveButton';
 import Image from 'next/image';
 import backIcon from '@/assets/BackArrow.svg';
 import Link from 'next/link';
+import { Bounce, toast, ToastContainer } from 'react-toastify';
+import useDarkMode from '@/hooks/useDarkMode';
+import 'react-toastify/dist/ReactToastify.css';
+import { validateEmail } from '@/lib/fieldValidator';
+import { updateUserEmail } from '@/hooks/apis/users';
+import { canParseJson } from '@/utils/helpers';
+import { generateTokenAndEmail } from '@/lib/generateVerification';
+import { useRouter } from 'next/navigation';
 
 const Editaccount: React.FC = () => {
-  const userEmail = 'Deerbuttcheeks@gmail.com';
+  const userEmail = 'new@example.com'; // Replace with actual user email from authentication state or context
```

Comment:
for this story, part of it is to load the text with the user's email, be sure to have this implemented as it is part of the logic

Reply:
As far as I know the email is not being saved in the session, if it is then can you pls help me out and show me how to fetch it, if not then that should be a different story

Reply:
This is on me for not updating on this, but you will not be fetching the email from the session as this is a security vulnerability, and it should not be accessed easily.

As an alternative you should save the UserID as either a cookie, or within the session (whichever is easier for you) and then fetch the user by ID. only the email should return through the network for now but with that you can then load it.

This is also a part of the story regarding the email log. But from here on, or at least after we deploy we should define a clear acceptance criteria going forward since there has been confusion with several stories on whats in or out of scope.

Reply:
when it is said that there should be logic with change email, that also means that it must change the email for the account. as it currently stands this cannot go to production because the edit account feature is non-functional. The point of the story when it says to integrate logic means not only using the hook, but successfully changing the email that I am currently logged into, this also goes for change password.

I also have made no mention of logging the session to be part of this story.

I understand u want this in main, but we need a fully complete edit account page, with the full logic integrated otherwise this story is not complete. It is not adding more to this story since that was the initial requirement to begin with. We are on a tight deadline yes, but that doesn't mean shipping incomplete features to rush to deploying to production.

Reply:
Well saving the user-email should be upon login, following that logic the Login story is incomplete and these two stories will be on hold until that story is. 
Also I meant putting it into main so we can create the test doc and having a close deadline for the other story. I will create that story and assign myself to it for now